<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlanOp2XLS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Import danych</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="wczytajPDFButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Wczytaj PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zapiszXLSButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Zapisz XLS</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="danePacjentowTable">
          <column>
           <property name="text">
            <string>Sala</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Oddział</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Lp</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pacjent</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PESEL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nr księgi głównej</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wiek</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Rozpoznanie</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Personel</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Zabieg</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Przypisanie lekarzy</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="dodajPrzypisanieButton">
              <property name="text">
               <string>Dodaj</string>
              </property>
              <property name="icon">
               <iconset theme="document-new">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="usunPrzypisanieButton">
              <property name="text">
               <string>Usuń</string>
              </property>
              <property name="icon">
               <iconset theme="document-delete">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="odswiezPrzypisaniaButton">
              <property name="text">
               <string>Odśwież</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="daneLekarzyTable"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Wydruki</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="templatkiTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="generujWydrukiButton">
            <property name="text">
             <string>Generuj wydruki</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="otworzTemplatkeButton">
            <property name="text">
             <string>Otwórz templatkę</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="otworzKatalogButton">
            <property name="text">
             <string>Otwórz katalog</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="odswiezTemplatkiButton">
            <property name="text">
             <string>Odśwież</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlanOp2XLS">
    <property name="title">
     <string>PlanOp2XLS</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionWyj_cie"/>
   </widget>
   <addaction name="menuPlanOp2XLS"/>
  </widget>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionWyj_cie">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
